pokemon/src/components/pokemon/PokemonCard.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import {Link} from 'react-router-dom';
+
+import styled from 'styled-components';
+
+// const Sprite = styled.img`
+//     width: 5em;
+//     height: 5em;
+// `;
+
+const Card = styled.div`
```

Comment:
Curious what you think of styled components? I've seen these used in a few areas and they look pretty handy.